scvi/autotune/_manager.py

```diff
@@ -21,6 +15,13 @@
 from ._types import TunableMeta
 from ._utils import in_notebook
 
+try:
+    from ray import air, tune
+    from ray.tune.integration.pytorch_lightning import TuneReportCallback
+except ImportError:
+    pass
+
+
 logger = logging.getLogger(__name__)
```

Comment:
I'm pretty sure this needs to remain where it was, as of the import issues we were having, right @martinkim0?

Reply:
Yup, ideally this should be where it was before.